bluebrain/repo-patches/packages/steps/package.py

```diff
@@ -75,9 +75,9 @@ class Steps(CMakePackage):
     depends_on("py-gcovr", when="+coverage", type="build")
     depends_on("py-matplotlib", type=("build", "test"))
     depends_on("py-mpi4py", when="+distmesh", type=("build", "test", "run"))
-    depends_on("py-nose", when="@3:", type=("build", "test"))
+    depends_on("py-nose", when="@3:4", type=("build", "test"))
```

Comment:
I found the dependency which forced the use of an old py-virtualenv! We don't need that anymore!